extensions/keycloak/runtime/src/main/java/io/quarkus/keycloak/KeycloakRecorder.java

```diff
@@ -33,6 +33,11 @@ public QuarkusDeploymentContext createKeycloakDeploymentContext(AdapterConfig de
                 config = loadConfig(getClass().getClassLoader());
             }
 
+            if (config == null) {
+                throw new IllegalStateException(
+                        "Keycloak configuration not found, please refer to the Quarkus Keycloak guide to configure your application");
```

Comment:
```suggestion
                        "Keycloak configuration not found, please refer to the Quarkus documentation to learn how to configure your application");
```

I would be a bit more vague to have a better chance to stay up to date. Could you adjust?

That  being said, I'm not very convinced by how this Keycloak configuration works. I'll take a look tomorrow and start a discussion on the list.

Reply:
Adjustment done.